### PR TITLE
Fix Polish language

### PR DESCRIPTION
**Pull request by zlElo**
Fixed polish language

### CODE REVIEW

**Comment by zlElo** [comment]
8 months later...

File: locales/pl-PL.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
   "Name": "Nazwa",
   "Username": "Nazwa użytkownika",
   "Password": "Hasło",
-  "Email": "E-mail",
+  "Email": "E-Mail",
```
Comment (alchaplinsky) [inline locales/pl-PL.json:85]: Hey @zlElo what's the point of this fix? I don't think I've ever seen this spelling with a capital `M`. It's usually either `E-mail` or `e-mail`

Comment (zlElo) [reply locales/pl-PL.json:85]: In Germany, we write every time E-Mail instead of E-mail.

Comment (alchaplinsky) [reply locales/pl-PL.json:85]: Right, so German locale has it spelled correctly https://github.com/swiftyapp/swifty/blob/master/locales/de-DE.json#L85 
Why changing Polish locale?

Comment (zlElo) [reply locales/pl-PL.json:85]: I know, I translated it into German 😂 But Okay, I understand... Then I close it

**Pull request closed without merging**